docs/reference/filters.md

```diff
@@ -80,6 +80,23 @@ foo3:
   -> <dynamic>;
 ```
 
+
+## blockContent
+
+Block a request based on it's body content.
+
+The filter max request body is 2MiB by default and can be overidden with `-max-matcher-buffer-size=<int>`.
```

Comment:
The sentence is not correct because it is only using a 2MiB buffer as default and does not stop processing the body.